sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po

```diff
@@ -1,22 +1,26 @@
-# Spanish translations for sphinx_rtd_theme.
-# Copyright (C) 2018 Read the Docs
+# English translations for sphinx_rtd_theme.
```

Comment:
This file is generated so no need to directly edit, but this line should read "Spanish translations" instead of English.

Reply:
Oh hrm. I'll poke this a little bit, I'm not sure exactly where this would be coming from. Ultimately, no one should see this, as we don't recommend manually editing the po files -- if someone put up a pr with po files, I'd redirect them to transifex anyways.

Reply:
Something automated overwrote this, manually editing it back seems to stick though -- through transifex update and extraction of messages